hack/cli-coexistence/scripts/setup_machine_with_legacy_tanzu_cli_v0_28.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# This script is used to setup/install old Tanzu CLI v0.28 on the machine
+echo This script is used to setup/install Tanzu CLI v0.28 on the machine
+
+# Tanzu config directory path to preset the config data
+TANZU_CONFIG_DIR=$1
+
+GOHOSTOS=$(go env GOHOSTOS)
+GOHOSTARCH=$(go env GOHOSTARCH)
+
+# ROOT_DIR of Docker image is /app
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" || exit; pwd)"
+
+# Remove old build
+ rm -rf "${ROOT_DIR:?}"/bin
+
+# Get Tanzu CLI v0.28 release build
+ wget -P "${ROOT_DIR:?}"/bin https://github.com/vmware-tanzu/tanzu-framework/releases/download/v0.28.1/tanzu-cli-"$GOHOSTOS"-"$GOHOSTARCH".tar.gz
```

Comment:
working on to get this build once on build rather than everytime this script is run